modules/schematics/src/effect/files/__name@dasherize@if-flat__/__name@dasherize__.effects.ts

```diff
@@ -1,10 +1,33 @@
 import { Injectable } from '@angular/core';
 import { Actions, Effect<% if (feature) { %>, ofType<% } %> } from '@ngrx/effects';
-<% if (feature) { %>import { <%= classify(name) %>ActionTypes } from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';<% } %>
+<% if (feature && !api) { %>import { <%= classify(name) %>ActionTypes } from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';<% } %>
+<% if (feature && api) { %>import { catchError, map, switchMap } from 'rxjs/operators';<% } %>
+<% if (feature && api) { %>import { Load<%= classify(name) %>sFailure, Load<%= classify(name) %>sSuccess, <%= classify(name) %>ActionTypes } from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';<% } %>
 
 @Injectable()
 export class <%= classify(name) %>Effects {
-<% if (feature) { %>
+<% if (feature && api) { %>
+
+  private _testObservable = of([
```

Comment:
What about using `EMPTY` instead?

Reply:
I am good with changing to the `EMPTY` constant from `rxjs`

Reply:
Few things:

- I suggest using concatMap instead of switchMap
- We don't need a private variable here, just inline the created observable inside the the *Map pipe. You're going to replace this with a service call. Maybe an observable of an empty array since we don't know what the state structure is.